app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = sum(
+        1 for friend in friends if not friend.get("wearing_a_mask", False)
+    )
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError as e:
+            return str(e)
+        except NotWearingMaskError:
+            pass
```

Comment:
using pass in exceptions is not the best practice
https://stackoverflow.com/questions/21553327/why-is-except-pass-a-bad-programming-practice